template/base/base.html

```diff
@@ -169,9 +155,8 @@ <h4>{{index .Map "page_name"}}</h4>
 
         <div class="heading-elements">
             <ul class="breadcrumb heading-text">
-                <li><a href="index.html"><i class="icon-home2 position-left"></i> Home</a></li>
-                <li><a href="1_col.html">Network</a></li>
-                <li class="active">VLANs</li>
+                <li><a href="/"><i class="icon-home2 position-left"></i> Home</a></li>
+                <li>{{index .Map "page_name"}}</li>
```

Comment:
does it only support one level breadcrumbs navigation for now? like "Home / <page_name>"

Reply:
Yup - only single level breadcrumbs for now. Currently only the current page is templated. Will need a lot more code in the db for multilayer breadcrumbs & figure out hierarchy for pages.